package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitfinex-api-node",
-  "version": "6.0.0",
+  "version": "6.0.1",
   "description": "Node reference library for Bitfinex API",
   "engines": {
     "node": ">=8.3.0"
```

Comment:
also we should update bfx-api-node-rest to v5.1.1 and bfx-api-node-models to v1.7.1